js/src/utils/toScopeState.test.js

```diff
@@ -0,0 +1,156 @@
+/**
+ * Internal dependencies
+ */
+import toScopeState from './toScopeState';
+
+const SCOPE = {
+	// Manage product listings and accounts for Google Shopping
+	CONTENT: 'https://www.googleapis.com/auth/content',
+	// Manage new site verifications with Google
+	SITE_VERIFICATION_VERIFY_ONLY:
+		'https://www.googleapis.com/auth/siteverification.verify_only',
+	// Manage AdWords campaigns
+	AD_WORDS: 'https://www.googleapis.com/auth/adwords',
+};
+
+describe( 'toScopeState', () => {
+	let contentScopes;
+	let siteVerificationScopes;
+	let gmcScopes;
+	let adsScopes;
+	let allScopes;
+
+	beforeEach( () => {
+		contentScopes = [ SCOPE.CONTENT ];
+		siteVerificationScopes = [ SCOPE.SITE_VERIFICATION_VERIFY_ONLY ];
+		gmcScopes = [ SCOPE.CONTENT, SCOPE.SITE_VERIFICATION_VERIFY_ONLY ];
+		adsScopes = [ SCOPE.AD_WORDS ];
+		allScopes = [
+			SCOPE.CONTENT,
+			SCOPE.SITE_VERIFICATION_VERIFY_ONLY,
+			SCOPE.AD_WORDS,
+		];
+	} );
+
+	it( 'should return an object structure containing three properties', () => {
+		const scopeState = toScopeState( false, [] );
+
+		expect( scopeState ).toHaveProperty( 'gmcRequired' );
+		expect( scopeState ).toHaveProperty( 'adsRequired' );
+		expect( scopeState ).toHaveProperty( 'glaRequired' );
+	} );
+
+	it( 'when the `scopes` parameter is not given, should get `false` for all results', () => {
+		expect( toScopeState( false ) ).toMatchObject( {
+			gmcRequired: false,
+			adsRequired: false,
+			glaRequired: false,
+		} );
+		expect( toScopeState( true ) ).toMatchObject( {
+			gmcRequired: false,
+			adsRequired: false,
+			glaRequired: false,
+		} );
+	} );
+
+	describe.each( [ [ false ], [ true ] ] )(
+		'For `gmcRequired`, if the parameter `adsSetupComplete` = %p',
```

Comment:
💅  ❔ I'd prefer to make the string connects with the `describe` in front, so that it reads like "describe the returned `gmcRequired` value...". 😄 

```suggestion
		'returned `gmcRequired` value when parameter `adsSetupComplete` = %p'
```

Reply:
Same as https://github.com/woocommerce/google-listings-and-ads/pull/1071#discussion_r740716477.